src/angular-business/notification-toast/notification-toast.ts

```diff
@@ -0,0 +1,319 @@
+      mergedConfig.announcementMessage = undefined;
+    }
+
+    return this.openFromComponent(this._simpleSnackBarComponent, mergedConfig);
+  }
+
+  /**
+   * Dismisses the currently-visible snack bar.
+   */
+  dismiss(): void {
+    if (this._openedNotificationRef) {
+      this._openedNotificationRef.dismiss();
+    }
+  }
+
+  ngOnDestroy() {
+    // Only dismiss the notification at the current level on destroy.
+    if (this._notificationRefAtThisLevel) {
+      this._notificationRefAtThisLevel.dismiss();
+    }
+  }
+
+  /**
+   * Attaches the snack bar container component to the overlay.
+   */
+  private _attachNotificationToastContainer(
+    overlayRef: OverlayRef,
+    config: SbbNotificationToastConfig
+  ): NotificationToastContainerComponent {
+    const userInjector = config?.viewContainerRef?.injector;
+    const injector = new PortalInjector(
+      userInjector || this._injector,
+      new WeakMap([[SbbNotificationToastConfig, config]])
+    );
+
+    const containerPortal = new ComponentPortal(
+      this._snackBarContainerComponent,
+      config.viewContainerRef,
+      injector
+    );
+    const containerRef: ComponentRef<NotificationToastContainerComponent> = overlayRef.attach(
+      containerPortal
+    );
+    containerRef.instance.config = config;
+    return containerRef.instance;
+  }
+
+  /**
+   * Places a new component or a template as the content of the snack bar container.
+   */
+  private _attach<T>(
+    content: ComponentType<T> | TemplateRef<T>,
+    userConfig?: SbbNotificationToastConfig
+  ): SbbNotificationToastRef<T | EmbeddedViewRef<any>> {
+    const config = { ...new SbbNotificationToastConfig(), ...this._defaultConfig, ...userConfig };
+    const overlayRef = this._createOverlay(config);
+    const container = this._attachNotificationToastContainer(overlayRef, config);
+    const notificationRef = new SbbNotificationToastRef<T | EmbeddedViewRef<any>>(
+      container,
+      overlayRef
+    );
+
+    if (content instanceof TemplateRef) {
+      const portal = new TemplatePortal(content, null!, {
+        $implicit: config.data,
+        notificationRef,
+      } as any);
+
+      notificationRef.instance = container.attachTemplatePortal(portal);
+    } else {
+      const injector = this._createInjector(config, notificationRef);
+      const portal = new ComponentPortal(content, undefined, injector);
+      const contentRef = container.attachComponentPortal<T>(portal);
+
+      // We can't pass this via the injector, because the injector is created earlier.
+      notificationRef.instance = contentRef.instance;
+    }
+
+    // Subscribe to the breakpoint observer and attach the notification-handset class as
+    // appropriate. This class is applied to the overlay element because the overlay must expand to
+    // fill the width of the screen for full width notifications.
+    this._breakpointObserver
+      .observe(Breakpoints.HandsetPortrait)
```

Comment:
breakpoint from material, shouldn't we use our own?

Reply:
Absolutely. Missed this.